e2e/cypress/tests/support/api/system.d.ts

```diff
@@ -122,7 +122,7 @@ declare namespace Cypress {
          *       // do something with config
          *   });
          */
-        apiGetConfig(): Chainable<AdminConfig>;
+        apiGetConfig(): Chainable<{config: AdminConfig}>;
```

Comment:
Reference PR [11254](https://github.com/mattermost/mattermost-webapp/pull/11254) had these changes. Adding them here until it is merged.